### ISSUE TITLE
Missing LICENSE

### ISSUE THREAD

**Issue by TechnologyClassroom** [nlpxucan/WizardLM#34]
I see you have no LICENSE file for this project. The default is copyright. The model checkpoints are mentioned with regard to matching LLaMA, but the code in this repository is not mentioned.

I would suggest releasing the code under the GPL-3.0-or-later or AGPL-3.0-or-later license so that others are encouraged to contribute changes back to your project.

**Comment by Ciaranwuk** [comment]
To add to this, This is a fine-tuning of a Llama model, which is a GPLv3 license. GPLv3 licenses require any modifications of their work to also carry a GPLv3 license, so actually this one should have a GPLv3 license. Could be added by just copying the license across from the Llama model github page

**Comment by haiatn** [comment]
I agree, I think it could help many if the license file was added and the terms of usage would be clearer
